README.rst

```diff
@@ -43,14 +43,30 @@ CMakePPLang can be included in a CMake project by adding:
    include(FetchContent)
    FetchContent_Declare(
       cmakepp_lang
-      GIT_REPOSITORY https://github.com/CMakePP/cmakepp_lang
+      GIT_REPOSITORY https://github.com/CMakePP/CMakePPLang
    )
    FetchContent_MakeAvailable(cmakepp_lang)
 
 to your ``CMakeLists.txt`` file. CMakePPLang will then be downloaded as part
 of the CMake configuration step. More detailed instructions, can be found
 `here <https://cmakepp.github.io/CMakePPLang/getting_started/obtaining_cmakepplang.html>`__.
 
+Information about pinning a specific version of CMakePPLang can be found
```

Comment:
@ryanmrichard Should we point to the documentation for version pinning or include the short blurb commented out below this to include the instructions explicitly in the README?